app/controllers/assignments_controller.rb

```diff
@@ -21,6 +21,8 @@ def new
   end
 
   def edit
+    @assignment = Assignment.find(params[:id])
```

Comment:
This is unneeded, as the load resource on line 3 handles it.

Reply:
All of the changes in the controllers where responses to 500 level errors I was hitting in certain views.  I say we do not include any of these. whatever the appropriate fix is, it belongs in another PR.